README.md

```diff
@@ -13,7 +13,7 @@
 1. Once you have an SSH key, clone the [sc1-data-format repository](https://github.com/badgerloop-software/sc1-data-format) to your computer. Make sure to clone it using SSH (when you go to copy the clone link, there will be an SSH option above the link).
 2. Next, `cd` into the `sc1-driver-io` repository and run `git submodule update --init`.
 
-### Running the Dashboard
+### Running the Dashboard with QMake
```

Comment:
This was part of the checklist of things. I think we should stick with one build system. If you want to move forward with cmake, let's remove qmake (this section of the readme and solar-car-dashboard.pro) so we don't have to maintain both.